### DISCUSSION TITLE
Time to Next Filament Change

### DISCUSSION THREAD

**Discussion by FormulaNerd3D** [j7126/OctoPrint-Dashboard#409] [Ideas]
Hello -
I really enjoy using the dashboard you created. I have found it very useful and enjoy the ability to customize it. I have had several prints with filament changes and some of them are long duration prints. On a print I am currently doing, the time till next change is like 600+ min (which is accurate). Would it be possible to have the time till next change be in hh:mm:ss if the time exceeds 60 min and 59 sec? Also, similar to print completion ETA, have an ETA for the filament change? Again, I really enjoy the dashboard!

**Comment by j7126** [comment]
That's a good idea, should be pretty simple. I'll try to add it this weekend if I have time and don't forget.
